qrcode_generator/README.md

```diff
@@ -0,0 +1,30 @@
+# QR Code Generator
+### Desktop App to convert anything to QR Code!
+
+#### Currently supports the following file types
+- Portable Document Format (PDF)
+- Microsoft Word
+- Microsoft Powerpoint
+- Images (JPEG, PNG)
+- Text/URL
+
+Text/URL are processed locally.
+Other file types require access to Google Drive where the files are uploaded.<br><br>
+
+## How to Run
+- Create a [Python Virtual Environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) (Recommended)
+- Download the following python modules
+  - qrcode
+  - tkinter
+  - Google Client Library
+
```

Comment:
Suggestion - Make a script file, which will install the required packages.
Advantages - Rather than installing packages separately, one command will do the work.